src/elysium/test/sigmaprimitives_tests.cpp

```diff
@@ -42,6 +42,24 @@ BOOST_AUTO_TEST_CASE(private_key)
     BOOST_CHECK_NE(key.randomness, randomness);
 }
 
+BOOST_AUTO_TEST_CASE(private_key_inf)
```

Comment:
IMO better to use a long name so it is easier to understand (e.g. `private_key_infinity`).